consumer/dragonboard820c/guides/setup-development-environment.md

```diff
@@ -0,0 +1,537 @@
+
+```shell
+sudo tasksel
+```
+
+This command provides a character based UI.
+Select the following by typing space.
+
+- Debian Desktop Environment
+- All or some indented GUI themes
+- SSH Server
+
+select sddm Desktop Manager when asked.
+  
+Reboot
+
+```shell
+reboot
+```
+
+## Setup linaro user
+
+```shell
+login linaro
+```
+
+Type linaro as a password.
+
+**Add Python to the PYTHONPATH environment variable.**
+
+```shell
+sudo sed -i '$a export PYTHONPATH=/usr/local/lib/python3.6/dist-packages:/usr/local/lib/python2.7/site-packages\n' ~/.bashrc
+```
+
+## Add a new user
+**SSH ACL**
+
+```shell
+sudo groupadd -rf ssh
+sudo usermod -a -G ssh root
+sudo adduser $USER ssh
+```
+
+**I²C ACL**
+
+```shell
+sudo adduser $USER i2c
+```
+
+**logout to root**
+
+```shell
+logout
+```
+
+**Fix /etc/adduser.conf before we add a new user**
+
+```shell
+sed -i 's/#ADD_EXTRA_GROUPS/ADD_EXTRA_GROUPS/' /etc/adduser.conf
+sed -i 's/#EXTRA_GROUPS="dialout cdrom floppy audio video plugdev users"/EXTRA_GROUPS="linaro adm dialout cdrom sudo audio dip video plugdev staff systemd-journal netdev bluetooth ssh pulse-access i2c"/' /etc/adduser.conf
+```
+
+**Add the new user**
+
+```shell
+adduser [new username] --add_extra_groups
+```
+Provide Full name and other info if you wish.
+
+```shell
+login [new username]
+```
+
+Type password
+
+let's test which group the user belongs to
+
+```shell
+groups
+```
+
+You should see this:
+
+```shell
+[new username] adm dialout cdrom sudo audio dip video plugdev staff systemd-journal netdev bluetooth ssh pulse-access i2c linaro
+```
+
+**Add Python to the PYTHONPATH environment variable.**
+
+```shell
+sudo sed -i '$a export PYTHONPATH=/usr/local/lib/python3.6/dist-packages:/usr/local/lib/python2.7/site-packages\n' ~/.bashrc
+```
+
+logout to root
+
+```shell
+logout
+```
+
+## Add SSH keys
+Please follow [these](https://www.ssh.com/ssh/putty/windows/puttygen) instructions to generate SSH private and public keys with PuttyGen on Windows.
+To be able to access the board via SSH properly we copy generated SSH private and public keys.
+
+```shell
+mkdir .ssh
+cd .ssh
+```
+
+**Copy Private Key**
+Fire up your favorite editor (nano) and copy the the content of id_rsa (private key).
+ 
+```shell
+nano id_rsa
+```
+
+**Copy Public Key**
+Copy the the content of id_rsa.pub (public key).
+
+```shell
+nano id_rsa.pub
+```
+
+**Copy Public Key**
+Copy the the content of authorized_keys.
+
+```shell
+nano authorized_keys
+```
+
+**Make these keys available for linaro and [new username]**
+
+```shell
+cd ..
+cp -r .ssh /home/linaro/
+cp -r .ssh /home/[new username]/
+```
+
+**Update ```/etc/ssh/sshd_config``` to enable SSH and X11 windowing system for root and members of the ssh group that linaro and [new username] are member of.** 
+
+```shell
+sed -i 's/#PermitRootLogin/PermitRootLogin/' /etc/ssh/sshd_config
+sed -i 's/PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config

```

Comment:
this is a bad security practice. you are writing a guide about IoT, i would prefer to don't encourage password authentication for root users and rely on SSH key only.

Reply:
Updated in the new guide